packages/shared/shared-types.d.ts

```diff
@@ -61,7 +61,6 @@ export interface DesktopSettingsType {
   syncAllAccounts: boolean
   /** @deprecated The last used file location for the save dialog is now only kept in memory and not persisted anymore between sessions. */
   lastSaveDialogLocation: string | undefined
-  experimentalEnableMarkdownInMessages: boolean
   enableWebxdcDevTools: boolean
```

Comment:
```suggestion
  /** @deprecated */
  experimentalEnableMarkdownInMessages?: boolean
  enableWebxdcDevTools: boolean
```

I think we should not remove the properties from the type definition in such a case, because in reality it still exists: we don't actually clean it up from storage.